src/driver.ts

```diff
@@ -24,7 +27,7 @@ export const findDevice = (): Device | null => {
   const matchingDevice = HID.devices().find(
     (device) =>
       device.vendorId === VENDOR_ID &&
-      device.productId === PRODUCT_ID &&
+      PRODUCT_IDS.includes(device.productId, 0) &&
```

Comment:
Is there a reason why we need the `0` param here?

Reply:
had a warning from IDE. but seems it works without it

Reply:
removed